skopt/utils.py

```diff
@@ -292,3 +295,50 @@ def cook_estimator(base_estimator, space=None, **kwargs):
 
     base_estimator.set_params(**kwargs)
     return base_estimator
+
+def transform_dims(dimensions, estimator="GP"):
+    """
+    Transforms space for Gaussian processes.
```

Comment:
We should definitely explain what the function does here instead of referring back to some historical setup in the code. I'd suggest something like:
```
Normalize all dimensions.

This is particularly useful for gaussian process based regressors and is used internally by `gp_minimize.`
```